resources/themes/cura-light/icons/plugin_48px.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
(Same issue as above: Don't name vector images after how many pixels they were designed for.)